python/ray/scripts/scripts.py

```diff
@@ -2223,6 +2222,84 @@ def global_gc(address):
     print("Triggered gc.collect() on all workers.")
 
 
+@cli.command(hidden=True)
+@click.option(
+    "--address", required=False, type=str, help="Override the address to connect to."
+)
+@click.option(
+    "--node-id",
+    required=True,
+    type=str,
+    help="Hex id of the worker node to be drained.",
+)
+@click.option(
+    "--reason",
+    required=True,
+    type=click.Choice(
+        # Exclude 0 which is UNSPECIFIED
+        [item[0] for item in autoscaler_pb2.DrainNodeReason.items() if item[1] != 0]
```

Comment:
use `DrainNodeReason.UNSPECIFIED`?

Reply:
wondering from a usability POV, we should make it less verbose: i.e. "idle" -> DrainNodeReason.DRAIN_NODE_REASON_IDLE_TERMINATION

Reply:
I think it's fine since the help message prints out all the valid strings and I don't need to change this code if we add new drain reasons.

Reply:
yea seems fine for internal/private API